css/contact-us.css

```diff
@@ -201,42 +263,90 @@ textarea.form-control__input {
   width: 1px;
 }
 
-.form-switcher__box span {
-  display: inline-block;
-  width: 10px;
-  height: 10px;
-  background: white;
-  border-radius: 50%;
+
+.form-switcher {
+  padding: 24px;
+  display: flex;
+  background-color: #1D1C28;
+  justify-content: space-between;
+}
+
+.form-switcher__text {
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 20px;
 }
 
 .form-switcher__box {
-  display: inline-block;
-  background: #4d4040;
+  display: block;
   width: 40px;
-  height: 20px;
-  border-radius: 6px;
+  height: 24px;
+  border-radius: 10px;
+  background-color: rgba(229, 236, 251, 0.36);
+  position: relative;
+  cursor: pointer;
+  transition: background-color .2s ease;
 }
 
-.form-switcher__input:checked + .form-switcher__box span {
-  transform: translateX(35px);
+.form-switcher__box::before {
+  content: '';
+  display: block;
+  position: absolute;
+  width: calc(100% + 6px);
+  height: calc(100% + 6px);
+  left: 50%;
+  top: 50%;
+  opacity: 0;
+  transform: translate(-50%, -50%);
+  transition: opacity .2s ease;
+  pointer-events: none;
+  border: 2px solid #3892F3;
+  border-radius: 100px;
 }
 
-.form-switcher__input:checked + .form-switcher__box {
-  background-color: blue;
+.form-switcher__box::after {
+  content: '';
+  width: 16px;
+  height: 16px;
+  border-radius: 50%;
+  background-color: #FFFFFF;
+  position: absolute;
+  top: 2px;
+  left: 2px;
+  transition-property: transform, background-color;
+  transition-duration: .2s;
 }
 
 
+.form-switcher__input:checked + .form-checkbox__box::before {
+  opacity: 1;
+}
+
+.form-switcher__input:disabled + .form-checkbox__box {
+  background-color: rgba(208, 215, 251, 0.17);
+  cursor: default;
+  opacity: .5;
+}
 
+.form-switcher__box::hover {
+  background-color: rgba(212, 217, 242, 0.26);
+}
 
+.form-switcher__input:checked + .form-checkbox__box::hover {
+  background-color: #3892F3;
+}
 
-.form-switcher {
-  padding: 24px;
-  display: flex;
-  background-color: ;
-  justify-content: space-between;
+.form-switcher__input:checked + .form-switcher__box {
+  background-color: blue;
 }
 
+#switch:checked + .form-switcher__box {
+  background-color: #0265DC;
+}
 
+#switch:checked + .form-switcher__box::before {
```

Comment:
To make "switch circle" working just replace ::before pseudo-element with ::after :)

Reply:
You also need to fix the class in HTML
form-swithcher__input ==> form-switcher__input

Reply:
done :-)

Reply:
Didn't change before to after)

Reply:
Cool, now switcher is working! But it still looks strange - the pin is outside...
It's up to you - to fix it or not, but I close this discussion.